test/lib/flat_mutation_reader_assertions.hh

```diff
@@ -554,6 +579,20 @@ private:
             return {rt.position(), {rt.tombstone().timestamp, {}}};
         }
     }
+    void reset_rtcs() {
+        _rtcs = {};
+    }
+    void add_rtc(const range_tombstone_change& rtc) {
+        if (rtc.tombstone()) {
+            if (_rtcs) {
+                _rtcs->tombstone() = rtc.tombstone();
+            } else {
+                _rtcs.emplace(rtc);
+            }
+        } else {
+            _rtcs.reset();
+        }
+    }
```

Comment:
I think it would be simpler to just keep the last tombstone instead of this complicated optional.